gitium/readme.txt

```diff
@@ -5,7 +5,7 @@ Donate link: https://www.presslabs.com/gitium/
 Tags: git, version, versioning, deployment, version-control, github, bitbucket, travis, code, revision, testing, development, branch, production, staging, debug, plugin, gitium, presslabs, simple
 Requires at least: 3.9
 Tested up to: 4.6
-License: GPLv2
+License: GPLv3
```

Comment:
It contains the same basic intent, but it brings improvements. It's the standard nowadays (since 2007) and why would you use a license written in 1991 instead of a newer one with the same purpose and intent? 

See: http://www.ifross.org/en/what-difference-between-gplv2-and-gplv3
even more: https://copyleft.org/guide/

**for short:** 
- internationalization: they used new terminology, rather than using language tied to US legal concepts
- patents: they specifically address patents
- “Tivo-ization”: they address the restrictions in consumer products that take away, though hardware, the ability to modify the software
- DRM: they address digital rights management (which they call digital restrictions management)
- compatibility: they addressed compatibility with some other open source licenses
- termination: they addressed specifically what happens if the license is violated and the cure of violations